docs.rs builds are failing for the 0.17.0-rc

docs.rs builds for many bevy_crates are failing on the 0.17.0-rc release.

```
# rustc version
rustc 1.91.0-nightly (5eda692e7 2025-09-11)# docs.rs version
docsrs 0.6.0 (f42173de 2025-09-12)# build log
[INFO] running `Command { std: "docker" "create" "-v" "/home/cratesfyi/workspace-builder/builds/bevy_ecs-0.17.0-rc.1/target:/opt/rustwide/target:rw,Z" "-v" "/home/cratesfyi/workspace-builder/builds/bevy_ecs-0.17.0-rc.1/source:/opt/rustwide/workdir:ro,Z" "-v" "/home/cratesfyi/workspace-builder/cargo-home:/opt/rustwide/cargo-home:ro,Z" "-v" "/home/cratesfyi/workspace-builder/rustup-home:/opt/rustwide/rustup-home:ro,Z" "-e" "SOURCE_DIR=/opt/rustwide/workdir" "-e" "CARGO_TARGET_DIR=/opt/rustwide/target" "-e" "DOCS_RS=1" "-e" "CARGO_HOME=/opt/rustwide/cargo-home" "-e" "RUSTUP_HOME=/opt/rustwide/rustup-home" "-w" "/opt/rustwide/workdir" "-m" "6442450944" "--cpus" "6" "--user" "1001:1001" "--network" "none" "ghcr.io/rust-lang/crates-build-env/linux@sha256:e90291280db7d1fac5b66fc6dad9f9662629e7365a55743daf9bdf73ebc4ea79" "/opt/rustwide/cargo-home/bin/cargo" "+nightly" "rustdoc" "--lib" "-Zrustdoc-map" "--all-features" "--config" "build.rustdocflags=[\"--cfg\", \"docsrs\", \"-Zunstable-options\", \"--generate-link-to-definition\", \"-Z\", \"unstable-options\", \"--emit=invocation-specific\", \"--resource-suffix\", \"-20250911-1.91.0-nightly-5eda692e7\", \"--static-root-path\", \"/-/rustdoc.static/\", \"--cap-lints\", \"warn\", \"--extern-html-root-takes-precedence\"]" "--offline" "-Zunstable-options" "--config=doc.extern-map.registries.crates-io=\"https://docs.rs/{pkg_name}/{version}/x86_64-unknown-linux-gnu\"" "-Zrustdoc-scrape-examples" "-j6" "--target" "x86_64-unknown-linux-gnu", kill_on_drop: false }`
[INFO] [stderr] WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
[INFO] [stdout] 72bc310e2316638d75b8b63407294e56158adf926c4e42803cd7b9d048c5954c
[INFO] running `Command { std: "docker" "start" "-a" "72bc310e2316638d75b8b63407294e56158adf926c4e42803cd7b9d048c5954c", kill_on_drop: false }`
[INFO] [stderr] warning: Rustdoc did not scrape the following examples because they require dev-dependencies: change_detection, events, resources
[INFO] [stderr]     If you want Rustdoc to scrape these examples, then add `doc-scrape-examples = true`
[INFO] [stderr]     to the [[example]] target configuration of at least one example.
[INFO] [stderr] warning: target filter specified, but no targets matched; this is a no-op
[INFO] [stderr]     Checking bevy_reflect v0.17.0-rc.1
[INFO] [stderr] error: Choosing a backend is required for automatic reflect registration. Please enable either the "auto_register_inventory" or the "auto_register_static" feature.
[INFO] [stderr]    --> /opt/rustwide/cargo-home/registry/src/index.crates.io-1949cf8c6b5b557f/bevy_reflect-0.17.0-rc.1/src/lib.rs:747:9
[INFO] [stderr]     |
[INFO] [stderr] 747 | / ...   compile_error!(
[INFO] [stderr] 748 | | ...       "Choosing a backend is required for automatic reflect registration. Please enable either the \"auto_register_inventory\" or the...
[INFO] [stderr] 749 | | ...   );
[INFO] [stderr]     | |_______^
[INFO] [stderr] 
[INFO] [stderr] error[E0425]: cannot find function `register_types` in module `crate::__macro_exports::auto_register`
[INFO] [stderr]    --> /opt/rustwide/cargo-home/registry/src/index.crates.io-1949cf8c6b5b557f/bevy_reflect-0.17.0-rc.1/src/type_registry.rs:158:48
[INFO] [stderr]     |
[INFO] [stderr] 158 |         crate::__macro_exports::auto_register::register_types(self);
[INFO] [stderr]     |                                                ^^^^^^^^^^^^^^ not found in `crate::__macro_exports::auto_register`
[INFO] [stderr] 
[INFO] [stderr] For more information about this error, try `rustc --explain E0425`.
[INFO] [stderr] error: could not compile `bevy_reflect` (lib) due to 2 previous errors
[INFO] running `Command { std: "docker" "inspect" "72bc310e2316638d75b8b63407294e56158adf926c4e42803cd7b9d048c5954c", kill_on_drop: false }`
[INFO] running `Command { std: "docker" "rm" "-f" "72bc310e2316638d75b8b63407294e56158adf926c4e42803cd7b9d048c5954c", kill_on_drop: false }`
[INFO] [stdout] 72bc310e2316638d75b8b63407294e56158adf926c4e42803cd7b9d048c5954c
```

--- a/crates/bevy_anti_alias/Cargo.toml
+++ b/crates/bevy_anti_alias/Cargo.toml
@@ -43,3 +43,4 @@ workspace = true
 [package.metadata.docs.rs]
 rustdoc-args = ["-Zunstable-options", "--generate-link-to-definition"]
 all-features = true
+features = ["bevy_image/zstd_rust"]

--- a/crates/bevy_app/Cargo.toml
+++ b/crates/bevy_app/Cargo.toml
@@ -115,3 +115,4 @@ workspace = true
 [package.metadata.docs.rs]
 rustdoc-args = ["-Zunstable-options", "--generate-link-to-definition"]
 all-features = true
+features = ["bevy_reflect/auto_register_static"]

--- a/crates/bevy_core_pipeline/Cargo.toml
+++ b/crates/bevy_core_pipeline/Cargo.toml
@@ -48,3 +48,4 @@ workspace = true
 [package.metadata.docs.rs]
 rustdoc-args = ["-Zunstable-options", "--generate-link-to-definition"]
 all-features = true
+features = ["bevy_image/zstd_rust"]

--- a/crates/bevy_ecs/Cargo.toml
+++ b/crates/bevy_ecs/Cargo.toml
@@ -152,3 +152,4 @@ workspace = true
 [package.metadata.docs.rs]
 rustdoc-args = ["-Zunstable-options", "--generate-link-to-definition"]
 all-features = true
+features = ["bevy_reflect/auto_register_static"]

--- a/crates/bevy_pbr/Cargo.toml
+++ b/crates/bevy_pbr/Cargo.toml
@@ -84,3 +84,4 @@ workspace = true
 [package.metadata.docs.rs]
 rustdoc-args = ["-Zunstable-options", "--generate-link-to-definition"]
 all-features = true
+features = ["bevy_image/zstd_rust"]

